### ISSUE TITLE
Multi-person collaborative labeling

### ISSUE THREAD

**Issue by WenNicholas** [chakki-works/doccano#352]
your product is really great! I would like to consult: I would like to cooperate with the multi-person cooperation and assign the labeling task. Can you tell me more about this? Is everyone going to register an account, or just one person to register an account, and multi-person collaboration mark under the same account? Looking forward to your reply.

**Comment by icoxfog417** [comment]
[Please refer FAQ page](https://github.com/chakki-works/doccano/wiki/Frequently-Asked-Questions#i-want-to-add-annotators).

**Comment by WenNicholas** [comment]
Thank you for your reply, according to the link you gave, follow the above operation, create two new users, create a new project, these two steps are completed. But found that two accounts can not log in at the same time, and the last step Edit your project setting In the middle, I don't see the checkbox for the collaborative annotation, only the check box for randomize document order.
          After logging in to these two users, I did not find the project I just created, and I did not find the login super administrator account.
         I don't know what the definition of multiplayer collaboration is? Is it possible for multiple people to log in to the same account for data annotation, or for multi-person accounts to perform data annotation under the same task.
         Looking forward to your reply.
